tests/parser_templates/cli/show_interfaces/main.yaml

```diff
@@ -18,9 +18,9 @@
       - interfaces['GigabitEthernet0/0/1']['admin-status'] == 'enabled'
       - interfaces['Tunnel0']['admin-status'] == 'enabled'
       - interfaces['Tunnel0']['oper-status'] == 'down'
-      - interfaces['GigabitEthernet0']['counters']['in-broadcast-pkts'] == '4891144'
+      - interfaces['GigabitEthernet0']['counters']['in-broadcast-pkts'] | int == 4891144
```

Comment:
There shouldn't be a need to cast here, if you revert this change, it will work.